README.md

```diff
@@ -1,16 +1,29 @@
-# Charm for MongoDB on Kubernetes
+# Charmed MongoDB on Kubernetes
+## Overview
 
-## Description
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
```

Comment:
you may want to clarify what "day 0 to day 2" means, or link to a clarification - as written, it implies that you're on your own after three days.

Reply:
I also got tripped up on this awhile ago. Not the biggest fan, but it is a phrase that you'll see all over Canonicals docs -- and all over the software world. 

It is defined as "Day 0, Day 1, and Day 2 refer to different phases of the software life cycle. Day 0 is the design phase, Day 1 is where things get creative, and Day 2 is just a daily operations routine."

Reply:
I would but the link but I am worried about the link changing over time . There is no wiki for this defintion so the other links might change over time